docker/gaffer-jhub-options-server/src/gaffer-graphs.js

```diff
@@ -0,0 +1,330 @@
+
```

Comment:
Usually application code like this would live in [docker-tools](https://github.com/gchq/gaffer-tools) and only the docker/kubernetes config would live in this repo. Are we sticking to that policy or does it make sense to have the options server source code live here too?

Reply:
The options server code only exists for this specific use case, so it is okay for it to live here. Splitting it out into gaffer-tools would just complicate things for no reason.